src/gauging.jl

```diff
@@ -26,15 +27,13 @@ function vidal_gauge(
     vsrc, vdst = src(e), dst(e)
     ψvsrc, ψvdst = ψ_vidal[vsrc], ψ_vidal[vdst]
 
-    s1, s2 = find_subgraph((vsrc, 1), mts), find_subgraph((vdst, 1), mts)
-    edge_ind = commoninds(ψ_vidal[vsrc], ψ_vidal[vdst])
+    pe = partitionedge(pψψ, NamedEdge((vsrc, 1) => (vdst, 1)))
```

Comment:
Can this work if we drop of `NamedEdge` constructor? Maybe would require a generalization of `partitionedge` in `NamedGraphs`, but ideally anywhere an `AbstractEdge` is accepted, we can also use a `Pair`.

Reply:
Yeah, we should add a definition like that.